.gitignore

```diff
@@ -9,3 +9,4 @@ yarn-error.log
 /build/
 
 .DS_Store
+.vscode/
```

Comment:
This shouldn't be in the repo. This is a user-specific configuration and should therefore be in the global gitignore.
https://davidwalsh.name/global-gitignore

Reply:
Ohh, shit, there is already .idea, did I push that?

Reply:
No, I added .idea, vscode probably came from the tool. I'm cool with un-ignoring this!